actors/runtime/src/test_utils.rs

```diff
@@ -500,6 +500,9 @@ impl MockRuntime {
         })
     }
 
+    #[allow(dead_code)]
+    pub fn expect_abort(&mut self) {}
```

Comment:
This should be removed the free function at line 359 does this function for the mock runtime.